js/categoryDrink.js

```diff
@@ -0,0 +1,49 @@
+let catDrink=document.getElementsByClassName('categories-drink')
+let catDrinkName=[]
+fetch('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
```

Comment:
Same for Food categories, try to ask if data is in local storage or not, and if it is in local storage get them from it, not from the API

Reply:
done